### ISSUE TITLE
Manually curated training set access

### ISSUE THREAD

**Issue by ptynecki** [Adrian-Cantu/PhANNs#2]
Hello Adrian,

You and your team made a fantastic work with PhANNs. I like the methodology which you adopted and I hope to see newer extended version soon.

I'm trying to resolve bacteriophages structural proteins classification with different approach for feature engineering. Because of it, I would like to compare both our tools with the same training and testing datasets.

I would like to ask if you are planning to share training set including more than 80k (manually curated) protein samples which cover 11 fallowing structural classes?

Best regards,
Piotr

**Comment by Adrian-Cantu** [comment]
Hi Piotr,

The sequences used during training and testing can be found [here](https://edwards.sdsu.edu/phanns/download/expandedDB.tgz). The fasta files numbered 1-10 were used in the 10-fold cross-validation and 11 is the test set. 

Regards,
Adrian

**Comment by ptynecki** [comment]
@Adrian-Cantu 

Thanks for the comment Adrian.

Let me clarify one thing - did you use all the samples from these 10-subsets for training? After summing them into one piece we have more than **490k samples**.

I'm asking about this because on the [PhANNs about page](https://edwards.sdsu.edu/phanns/about) there is a information:

> PhANNs was trained against a database of 70,601 manually curated phage protein sequences.

Would you be able to share **PhANNs_DB.fasta.tgz** file which is linked to it?

Similar message we can catch on [webserver README](https://github.com/Adrian-Cantu/PhANNs/tree/master/web_server):

> It was trained on a manually curated database of more than 80,000 phage proteins. 

I will be very grateful if I could use exactly the same set of training samples in my research. After that, I will share with you the results immediately.
